cypress/integration/data-import/match-on-location.spec.js

```diff
@@ -0,0 +1,353 @@
+  ];
+
+  const jobProfileForCreate = {
+    profile: {
+      name: jobProfileName,
+      dataType: 'MARC'
+    },
+    addedRelations: [],
+    deletedRelations: []
+  };
+
+  // profiles for updating instance, holdings, item
+  const collectionOfMatchProfiles = [
+    {
+      matchProfile: { profileName: instanceHridMatchProfile,
+        incomingRecordFields: {
+          field: '001',
+        },
+        existingRecordFields: {
+          field: '001',
+        },
+        matchCriterion: 'Exactly matches',
+        existingRecordType: 'MARC_BIBLIOGRAPHIC' }
+    },
+    {
+      matchProfile: { profileName: holdingsPermLocationMatchProfile,
+        incomingRecordFields: {
+          field: '960',
+          subfield: 'a'
+        },
+        matchCriterion: 'Exactly matches',
+        existingRecordType: 'HOLDINGS',
+        holdingsOption: NewMatchProfile.optionsList.holdingsPermLoc }
+    },
+    {
+      matchProfile: { profileName: itemPermLocationMatchProfile,
+        incomingRecordFields: {
+          field: '960',
+          subfield: 'b'
+        },
+        matchCriterion: 'Exactly matches',
+        existingRecordType: 'ITEM',
+        itemOption: NewMatchProfile.optionsList.itemPermLoc }
+    },
+  ];
+
+  const holdingsUpdateMappingProfile = {
+    name: holdingsMappingProfile,
+    typeValue : NewFieldMappingProfile.folioRecordTypeValue.holdings
+  };
+
+  const itemUpdateMappingProfile = {
+    name: itemMappingProfile,
+    typeValue : NewFieldMappingProfile.folioRecordTypeValue.item
+  };
+
+  const holdingsUpdatesActionProfile = {
+    typeValue: NewActionProfile.folioRecordTypeValue.holdings,
+    name: holdingsActionProfile,
+    action: 'Update (all record types except Orders, Invoices, or MARC Holdings)'
+  };
+
+  const itemUpdatesActionProfile = {
+    typeValue: NewActionProfile.folioRecordTypeValue.item,
+    name: itemActionProfile,
+    action: 'Update (all record types except Orders, Invoices, or MARC Holdings)'
+  };
+
+  const jobProfileNameForUpdate = {
+    ...NewJobProfile.defaultJobProfile,
+    profileName: jobProfile,
+    acceptedType: NewJobProfile.acceptedDataType.marc
+  };
+
+  before(() => {
+    cy.loginAsAdmin();
+    cy.getAdminToken();
+
+    testData.jobProfileForCreate = jobProfileForCreate;
+
+    testData.forEach(specialPair => {
+      cy.createOnePairMappingAndActionProfiles(specialPair.mappingProfile, specialPair.actionProfile).then(idActionProfile => {
+        cy.addJobProfileRelation(testData.jobProfileForCreate.addedRelations, idActionProfile);
+      });
+    });
+    DataImportSettingsJobProfiles.createJobProfileApi(testData.jobProfileForCreate)
+      .then((bodyWithjobProfile) => {
+        testData.jobProfileForCreate.id = bodyWithjobProfile.body.id;
+      });
+
+    // upload a marc file for creating of the new instance, holding and item
+    cy.visit(TopMenu.dataImportPath);
+    DataImport.uploadFile('matchOnLocation.mrc', marcFileForCreate);
```

Comment:
A fixture file could not be found at any of the following paths:

    > cypress\fixtures\matchOnLocation.mrc
    > cypress\fixtures\matchOnLocation.mrc.[ext]

Reply:
Done